Router/Router/Controllers/LoginViewController.swift

```diff
@@ -0,0 +1,47 @@
+//  ViewController.swift
+//  Router
+//  Created by angelina on 27.12.2022.
+
+import UIKit
+/// Экран входа
+final class LoginViewController: UIViewController {
+    
+    // MARK: - Constants
+    private enum Constants {
+        static let login = "admin"
+        static let password = "123456"
+        static let userDefaultsKeyString = "isLogin"
+    }
+    
+    // MARK: - Private IBOutlet
+    @IBOutlet private weak var loginView: UITextField!
```

Comment:
```suggestion
    @IBOutlet private weak var loginTextField: UITextField!
```
С меня пример берешь?))

Reply:
ага) списываю